frontend/src/pages/onboarding/AvatarForm.tsx

```diff
@@ -11,15 +12,20 @@ export const AvatarForm = () => {
   const { setValue } = useFormContext<OnboardingFormValues>();
   const handleDrop = useCallback(
     (acceptedFiles: File[]) => {
-      const file = acceptedFiles[0];
+      let file = acceptedFiles[0];
 
       if (file) {
-        setValue(
-          'avatar',
-          Object.assign(file, {
-            preview: URL.createObjectURL(file),
-          })
-        );
+        new Compressor(file, {      
+          quality: 0.4,
+          success: (compressedResult: File) => {
```

Comment:
This quality value is relative?

I'm wondering if can set it to a fixed value. Would be better in case user uploads a very high res image.

Reply:
shld be % of compression, let's review this in the future